scripts/ccc

```diff
@@ -156,9 +168,9 @@ def main():
     command = sys.argv[1]
     dispatch = {'capture': lambda: capture(),
                 'uninstall': lambda: uninstall(sys.argv[2]),
-                'login': lambda: login(sys.argv[2], sys.argv[3]),
+                'invalidate': lambda: stage_recover(),
                 'remote': lambda: remote_install(*sys.argv[2:5]),
-                'help': lambda: HELP_MSG,
+                'help': lambda: print(HELP_MSG),
```

Comment:
I think this is the first thing that makes this script explicitly `python2` incompatible...